crates/rspack_plugin_css/src/parser_and_generator/mod.rs

```diff
@@ -38,6 +38,9 @@ use crate::{
 static REGEX_IS_MODULES: LazyLock<Regex> =
   LazyLock::new(|| Regex::new(r"\.module(s)?\.[^.]+$").expect("Invalid regex"));
 
+static REGEX_IS_COMMENTS: LazyLock<Regex> =
+  LazyLock::new(|| Regex::new(r"/\*.*?\*/").expect("Invalid regex"));
```

Comment:
Can this match comments with multiple lines ?

```suggestion
  LazyLock::new(|| Regex::new(r"/\*[\s\S]*?\*/").expect("Invalid regex"));
```

Reply:
Thanks, I agree.